pulp_2to3_migration/app/plugin/rpm/xml_utils.py

```diff
@@ -234,6 +234,8 @@ def newpkgcb(pkgId, name, arch):
     cr.xml_parse_primary_snippet(primary_xml, pkgcb=pkgcb, do_files=False)
     cr.xml_parse_filelists_snippet(filelists_xml, newpkgcb=newpkgcb)
     cr.xml_parse_other_snippet(other_xml, newpkgcb=newpkgcb)
+    # keep only the last 10 changelogs
+    package.changelogs = package.changelogs[-10:]
```

Comment:
This could be more like `-getattr("KEEP_CHANGELOG_LIMIT", settings, 1))`

Reply:
I think for simplicity let's keep it hard-coded until anyone asks to change it.

Reply:
Also this way, it can be hard-coded if needed.